prime-angular-frontend/src/app/lib/modules/ngx-busy/busy.service.ts

```diff
@@ -18,13 +18,16 @@ export class BusyService {
   /**
    * @description
    * @param message the message to be displayed when a busy has a busy thing (subscription, promise, array of either... etc)
-   * @param request the observable that will run while the busy message and spinner are shown
+   * @param request the function or observable that will run while the busy message and spinner are shown
    * @returns OperatorFunction
    */
-  public showMessagePipe<T>(message: string, request: Observable<any>): OperatorFunction<T, Observable<any>> {
+  public showMessagePipe<T, R>(message: string, request: R): UnaryFunction<Observable<T>, Observable<any>> {
     return pipe(
       tap((_) => { this._message = message }),
```

Comment:
Don't use single line with braces `tap((_) => this._message = message)` similarly when clearing the message

Reply:
Removed, not sure why but the first time I tried without braces it complained, now it seems fine.